README.md

```diff
@@ -28,3 +28,31 @@ The hub is a long term data store.
 * Many messaging systems are difficult to cluster
 * Many messaging system require custom clients
 
+
```

Comment:
i'm a little confused about why so much of this is showing up as a diff since it wasn't touched in your branch and it was changed in master. What am I missing?

Reply:
nevermind, this shows up as outdated, so the diff as a whole doesn't show these changes...just that one commit.